tutorials/navigation/navigation_using_navigationagents.rst

```diff
@@ -283,6 +338,173 @@ The following sections provides script templates for nodes commonly used with Na
             func _on_velocity_computed(safe_velocity: Vector2):
                 linear_velocity = safe_velocity
 
+   .. tab:: 2D C#
+
+      .. tabs::
+
+         .. code-tab:: csharp Node2D
+
+            using Godot;
+
+            public partial class MyNode2D : Node2D
+            {
+                [Export] private float _movementSpeed = 4.0f;
```

Comment:
Since the member is exported, let's make it a public property instead.

```suggestion
                [Export]
                public float MovementSpeed { get; set; } = 4.0f;
```